resteasy-core-spi/src/main/java/org/jboss/resteasy/resteasy_jaxrs/i18n/LogMessages.java

```diff
@@ -325,6 +325,14 @@ public interface LogMessages extends BasicLogger
    @Message(id = BASE + 370, value = "Failed to load: %s")
    void failedToLoad(@Cause Throwable cause, String resourceName);
 
+   @LogMessage(level = Level.ERROR)
+   @Message(id = BASE + 375, value = "Handling an uncaught exception with the default exception mapper.")
```

Comment:
I wonder if making this message so long has value. My point is that the important information for the user is that an error occurred. Do you really care about all the details?

Note that I'm not very familiar with RESTEasy/JAX-RS so it might actually have value.

Reply:
Maybe it doesn't make much sense actually. Maybe the message really should just be `cause.getLocalizedMessage()` in this case.

Reply:
I changed it a bit, but ended up keeping some kind of message indicating it came from RESTEasy.